README.md

```diff
@@ -1,2 +1,2 @@
 # SimpleFSDB
-simple file system database 
+simple file system database 
```

Comment:
remove all .pyc. I recommend add `.gitignore` file and add `*.pyc`

Look at Mariam PR. she did the same